### ISSUE TITLE
Running bigscience/bloom-350m example returns AssertionError

### ISSUE THREAD

**Issue by cderinbogaz** [microsoft/DeepSpeed-MII#52]
When I run the following example from the readme:
```
import mii
mii_configs = {"tensor_parallel": 1, "dtype": "fp16"}
mii.deploy(task="text-generation",
           model="bigscience/bloom-350m",
           deployment_name="bloom350m_deployment",
           mii_config=mii_configs)
```
It returns the following:
```
`[/usr/local/lib/python3.7/dist-packages/mii/utils.py](https://localhost:8080/#) in check_if_task_and_model_is_valid(task, model_name)
    108     assert (
    109         model_name in valid_task_models
--> 110     ), f"{task_name} only supports {valid_task_models}"
    111 
    112 

AssertionError: text-generation only supports....
```
Error. I suspect this is related to a change in model weights. 

Can you point me in the right direction?

And also, thanks for this amazing repo! Can't wait to use it 👍 💯

**Comment by mrwyattii** [comment]
@cderinbogaz thanks for checking out MII! It looks like HuggingFace recently removed the 350m version of the bloom model. Could you try with `model="bigscience/bloom-560m"`? If that works for you, I'll make a PR to update our examples (or you can contribute one if you like).

**Comment by cderinbogaz** [comment]
Hi @mrwyattii, thanks for the pointer.
Indeed, the model was removed from the huggingface hub. I opened up a pull request to fix it here: https://github.com/microsoft/DeepSpeed-MII/pull/53

Thanks again for this amazing repo 🚀

**Comment by cderinbogaz** [comment]
I realised opt examples also have an issue. When local_path is defined, it gives the following error:
`TypeError: deploy() got an unexpected keyword argument 'local_model_path'`

**Comment by mrwyattii** [comment]
Thanks @cderinbogaz I just updated the examples to fix this in #68
